functions/Remove-SqlOrphanUser.ps1

```diff
@@ -6,31 +6,34 @@ Drop orphan users with no existing login to map
 
 .DESCRIPTION
 An orphan user is defined by a user that does not have their matching login. (Login property = "").
-If exists a login to map the drop will not be performed unless you specify the -Force parameter.
-	
+If user is the owner of the schema with the same name and if if the schema does not have any underlying objects the schema will be dropped.
+If user owns more than one schema, the owner of the schemas that does not have the same name as the user, will be changed to 'dbo'. If schemas have underlying objects, you must specify the -Force parameter so the user can be dropped.
+If exists a login to map the drop will not be performed unless you specify the -Force parameter (only when calling from Repair-SqlOrphanUser.
+
 .PARAMETER SqlServer
 The SQL Server instance.
 
 .PARAMETER SqlCredential
 Allows you to login to servers using SQL Logins as opposed to Windows Auth/Integrated/Trusted. To use:
 
-$scred = Get-Credential, then pass $scred object to the -SqlCredential parameter. 
+$scred = Get-Credential, then pass $scred object to the -SqlCredential parameter.
 
 Windows Authentication will be used if SqlCredential is not specified. SQL Server does not accept Windows credentials being passed as credentials. To connect as a different Windows user, run PowerShell as that user.
 
 .PARAMETER Users
 List of users to remove
 
 .PARAMETER Force
+If exists any schema which owner is the User, this will force the change of the owner to 'dbo'.
 If exists a login to map the drop will not be performed unless you specify this parameter.
 
-.PARAMETER WhatIf 
-Shows what would happen if the command were to run. No actions are actually performed. 
+.PARAMETER WhatIf
```

Comment:
For now just keep it. Pester fails when help for this parameters does not exists